ui/components/multichain-accounts/add-wallet-modal/add-wallet-modal.test.tsx

```diff
@@ -1,13 +1,35 @@
 import React from 'react';
 import { screen, fireEvent } from '@testing-library/react';
 import { renderWithProvider } from '../../../../test/jest';
+import {
+  ADD_WALLET_PAGE_ROUTE,
+  CONNECT_HARDWARE_ROUTE,
+  IMPORT_SRP_ROUTE,
+} from '../../../helpers/constants/routes';
 import { AddWalletModal } from './add-wallet-modal';
 
+const mockHistoryPush = jest.fn();
+const mockOpenExtensionInBrowser = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useHistory: () => ({
+    push: mockHistoryPush,
+  }),
+}));
```

Comment:
Mocking react router dom useHistory and opening the extension in browser for back button and import hardware wallet which doesn't have a popup view